packages/agreement-process/src/resources/templates/documents/agreementContractTemplate.html

```diff
@@ -118,7 +126,16 @@ <h2>Registrazione</h2>
         <strong>{{submissionTime}}</strong>, l’Infrastruttura ha trasmesso
         all’erogatore la richiesta di fruizione.
       </div>
-
+      {{#if delegationId}}
+      <div>
+        In data <strong>{{activationDate}}</strong> alle ore
+        <strong>{{activationTime}}</strong>, l’infrastruttura ha ricevuto
+        la dichiarazione di accettazione della richiesta di fruizione
+        (di seguito “dichiarazione di accettazione”) inviata dal delegato all’erogazione,
+        tramite l’operatore amministrativo con identificativo Area Riservata
+        <strong>{{delegateId}}</strong>.
+      </div>
+      {{else}}
```

Comment:
In the [mockup](https://www.figma.com/design/QlabxYPLjCE0QZmc7xtbw7/Interop-%E2%80%94-v0.11-LLGG-2024-Vol.-II?node-id=15236-25252&t=M4t7S7gC1wyQfd64-4), there seem to be more places where we should mention the delegate/delegator instead of the producer, while here I see that you show "delegato all'erogazione" only in this sentence related to the activation

<img width="501" alt="image" src="https://github.com/user-attachments/assets/e428e7e9-2052-4779-8af1-e988fee52c01">

Reply:
We will resolve that after develop merge, [PIN](https://pagopa.atlassian.net/browse/PIN-5774)